releasenotes/notes/43765.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+  - 43765
+releaseNotes:
+  - |
+    **Improved** the `istioctl pc secret` output to include certificate serial number in HEX.
```

```suggestion
    **Improved** the `istioctl pc secret` output to display the certificate serial number in HEX.
```
I first read include to mean that it would be in addition to the number. I'll agree with the change as an improvement, as I can't come up with a better action.

Only concern is if someone is scrapping the number and not expecting hex. Probably a very small number.

---

I thought about that -- but they'd have to scrape it from the default output format which would be odd; my assumption was if anyone is scraping anything from the output they are probably using yaml/json, but this change doesn't affect that.

---

IMO this should be good for the default output.